src/main/scala/org/broadinstitute/dsde/workbench/sam/config/GoogleServicesConfig.scala

```diff
@@ -31,6 +31,11 @@ final case class GoogleServicesConfig(
     groupSyncTopic: String,
     groupSyncSubscription: String,
     groupSyncWorkerCount: Int,
```

Comment:
Should there be a `disableUsersPubSubProject` value here too?

Reply:
So, it wasn't used. I did a bit of searching though, and the only usage of the `groupSyncPubSubProject` was in setting up the `googlePubSubDao`, so maybe we should do a refactor of that in general.